packages/at_persistence_secondary_server/test/at_notification_test.dart

```diff
@@ -18,5 +18,20 @@ void main() async {
       sleep(Duration(milliseconds: 50));
       expect(atNotification.isExpired(), false);
     });
+
+    test('test notificationExpiry time', () {
+      final builder = AtNotificationBuilder();
+      final notification = builder.build();
+      var notifExpiresAt = DateTime.now().toUtc().add(Duration(minutes: 15));
+      //when ttl is not passed, 15-mins is used as the default ttl
+      //notification.expiresAt and notifExpiresAt have the difference of a
+      // couple of milli seconds and they cannot asserted to be equal
+      expect(notification.expiresAt?.year, notifExpiresAt.year);
+      expect(notification.expiresAt?.month, notifExpiresAt.month);
+      expect(notification.expiresAt?.day, notifExpiresAt.day);
+      expect(notification.expiresAt?.hour, notifExpiresAt.hour);
+      expect(notification.expiresAt?.minute, notifExpiresAt.minute);
+      expect(notification.expiresAt?.second, notifExpiresAt.second);
```

Comment:
As an alternative, why not replace all of the above assertions with just one assertion that the milisecondSinceEpoch of both are within some number of milliseconds of each other? Seems like this test could flake if the second ticks over during the test

Reply:
this change has been made as part of [7dc827f](https://github.com/atsign-foundation/at_server/pull/1241/commits/7dc827f5a9ef7b2c738c0ccf78c45a29c114a368)